core/app_modules/cal.js

```diff
@@ -20,7 +20,7 @@ module.exports.addCalendarEvent = co.wrap(function*(username, event, user){
     }
   }
   if (!isOverlap) {
-    yield client.nonQuery('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);
+    yield client.query('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);

```

Comment:
Everything looks good! Looks like the backend is catching non-admins as well, so things should be even less of a problem. Feel free to merge it in.